lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -804,32 +804,34 @@ static struct nrf_sock_ctx *find_ctx(int fd)
 static void pollcb(struct nrf_pollfd *pollfd)
 {
 	int flags;
-	unsigned int signaled;
 	struct nrf_sock_ctx *ctx;
 
 	ctx = find_ctx(pollfd->fd);
 	if (!ctx) {
 		return;
 	}
 
-	k_poll_signal_check(&ctx->poll, &signaled, &flags);
```

Comment:
this can't be signaled at this point, so I removed this check and zero `flag` instead